CMake/cdat_modules/x264_deps.cmake

```diff
@@ -1 +1 @@
-set(X264_DEPS "")
+set(X264_deps ${pkgconfig_pkg})
```

Comment:
The reason it's "dependent" on pkgconfig is because it fixes a weird parallelism issue that @dnadeau4 was having (where the linker got confused as `libx264` and `pkgconfig` built at the same time).